settings.gradle

```diff
@@ -138,9 +138,9 @@ includeProject(":appcompat:appcompat-resources", "appcompat/appcompat-resources"
 includeProject(":appsearch:appsearch", "appsearch/appsearch", [BuildType.MAIN])
 includeProject(":appsearch:appsearch-compiler", "appsearch/compiler", [BuildType.MAIN])
 includeProject(":appsearch:appsearch-local-storage", "appsearch/local-storage", [BuildType.MAIN])
-includeProject(":arch:core-common", "arch/core-common", [BuildType.MAIN])
```

Comment:
i don't think we should make this change. it is basically giving hte module a weird name w/ a dot.

Reply:
How about arch:core:core-*?